plugins/inputs/socket_listener/README.md

```diff
@@ -77,6 +77,40 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## Content encoding for message payloads, can be set to "gzip" to or
   ## "identity" to apply no encoding.
   # content_encoding = "identity"
+
+  ## Packet splitting strategy and corresponding settings.
+  ## Available strategies are:
+  ##   newline         -- split at newlines (default)
+  ##   null            -- split at null bytes
+  ##   delimiter       -- split at delimiter byte-sequence in hex-format
+  ##                      given in `splitting_delimiter`
+  ##   fixed length    -- split after number of bytes given in `splitting_length`
+  ##   variable length -- split depending on length information received in the
+  ##                      data. The length field information is specified in
+  ##                      `splitting_length_field`.
+  # splitting_strategy = "newline"
+
+  ## Delimiter used to split received data to messages consumed by the parser.
+  ## The delimiter is a byte-sequence marking the end of a message
+  ## e.g. "0x0D0A" marks a Windows line-break (CR LF).
+  ## Note: This setting is only used for splitting_strategy = "delimiter".
+  # slitting_delimiter = ""
```

Comment:
```suggestion
  # splitting_delimiter = ""
```

Reply:
I think this one still needs to be updated, right?